en/release-notes/upgrade-guide-v2.1.md

```diff
@@ -14,67 +14,67 @@ This is not the first time __Cocos__ has upgraded the engine from 2D to 3D. A fe
 
 You can watch switching to **3D Camera** in action:
 
 ![](upgrade-guide-v2.1/641.gif).
 
-**3D model import**
+### 3D model import
 
-Currently, __Cocos Creator__ supports importing the **.fbx** format of 3D models. **.fbx** is the de-facto standard in 3D modeling software. The process of importing is very simple, just drag the **.fbx** model resources into the **Explorer** and wait a few moments for the import to complete. After the import is completed, you can see that the imported model is a folder that can be expanded in **Assets**. When importing the model, the editor will automatically parse the content of the model and generates resources such as __Prefab Meshes__ and __Skeletal Animations__ and so on. See [Import Models](../3d/import-model.html)
+Currently, __Cocos Creator__ supports importing the **.fbx** format of 3D models. **.fbx** is the de-facto standard in 3D modeling software. The process of importing is very simple, just drag the **.fbx** model resources into the **Assets** and wait a few moments for the import to complete. After the import is completed, you can see that the imported model is a folder that can be expanded in **Assets**. When importing the model, the editor will automatically parse the content of the model and generates resources such as __Prefab Meshes__ and __Skeletal Animations__ and so on. See [Import Models](../3d/import-model.md)
 
-![](upgrade-guide-v2.1/s_D5EB83DE4EB14ED7F4E0412694594BEA563C6F613AB6CAC04191666891BDD83C_1543326015234_image.png)
+![](upgrade-guide-v2.1/import-model.png)
 
-**Mesh Renderer Component**
+### Mesh Renderer Component
 
-Mesh Renderer is used to draw grid resources. See [Mesh Renderer](../3d/mesh-renderer.html).
+Mesh Renderer is used to draw grid resources. See [Mesh Renderer](../3d/mesh-renderer.md).
 
 ![](../3d/img/mesh_renderer.png)
 
-**Bone Animation Component**
+### Skeletal Animation Component
 
-The **Skeletal Animation** component inherits from the **Animation** component. Using the **Skeletal Animation** component is not much different from how you would use the **Animation** component. The exception is the **Skeletal Animation** component uses a clip that can only be a **Skeletal Animation** clip. The **Skeletal Animation** component is automatically added to the model's Prefab when it is imported. See [Bone Animation Component](../3d/skeleton-animation.html) for details.
+The **Skeletal Animation** component inherits from the **Animation** component. Using the **Skeletal Animation** component is not much different from how you would use the **Animation** component. The exception is the **Skeletal Animation** component uses a clip that can only be a **Skeletal Animation** clip. The **Skeletal Animation** component is automatically added to the model's Prefab when it is imported. See [Bone Animation Component](../3d/skeleton-animation.md) for details.
```

Comment:
Bone Animation Component -> Skeletal Animation Component

Reply:
oh, I missed that. I apologize. Silly.